modules/ROOT/pages/upgrade.adoc

```diff
@@ -3,89 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-== Before You Begin
-Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.
```

Comment:
Before the upgrade is ran, the requirements need to be verified against the requirements for the new version as sometimes there are small changes (usually opening new ports, sometimes adding a new port in the load balancer).
The recommendation is to re-verify that the environment is ready against the requirements for the new version.

Reply:
Added new wording.